servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ClientParentChannelInitializer.java

```diff
@@ -37,8 +38,7 @@ final class H2ClientParentChannelInitializer implements ChannelInitializer {
 
     H2ClientParentChannelInitializer(final H2ProtocolConfig config) {
         this.config = config;
-        final io.servicetalk.http.api.Http2Settings h2Settings = config.initialSettings();
-        nettySettings = toNettySettings(h2Settings);
+        nettySettings = toNettySettings(config.initialSettings()).pushEnabled(false).maxConcurrentStreams(0L);
```

Comment:
If we set `pushEnabled` to false we don't really need to set `maxConcurrentStreams`

Reply:
Either way it doesn't hurt. This is what grpc-java does. Let's be consistent.